### PR TITLE
Fix up the beez3 template.js

### PR DESCRIPTION
**Pull request by okonomiyaki3000**
Pull Request for Issue # .

### Summary of Changes

Don't use unnecessary closure, just use jQuery ready function.
Use javsacript strict mode
use event delegation where possible
Make tooltips and button groups work properly with repeatable subforms

### Testing Instructions

Use beez3. 
Create a page with a repeatable subform containing a radiobutton field with the btn-group class.

### Expected result

Expect newly created rows to initialize btn-groups properly.

### Actual result

Yes, it works.

### Documentation Changes Required

No.

### CODE REVIEW

**Comment by okonomiyaki3000** [comment]
This basically an alternative to #19722 with additional improvements and cleanup.

File: templates/beez3/javascript/template.js [hunk 1]
```diff
@@ -6,42 +6,64 @@
...
 
-(function($)
-{
-	$(document).ready(function()
+jQuery(function($) {
+	'use strict';
+
+	$(document).on('click', ".btn-group label:not(.active)", function() {
+			var $label = $(this);
+			var $input = $('#' + label.attr('for'));
```
Comment (ReLater) [inline templates/beez3/javascript/template.js]: Missing "$" before "label". Must be `var $input = $('#' + $label.attr('for'));`.

**Comment by ReLater** [comment]
I have tested this item :red_circle: unsuccessfully on 8157f499d648ee89af673b473b2d8d5c2ebc8bce<br /><br />Fails with error:
ReferenceError: label is not defined template.js:14:8<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/20225">issues.joomla.org/tracker/joomla-cms/20225</a>.</sub>

**Comment by ReLater** [comment]
I have tested this item :white_check_mark: successfully on db22fd0bdc9108b848364a4d195f50386c5e3fc3<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/20225">issues.joomla.org/tracker/joomla-cms/20225</a>.</sub>

**Comment by ReLater** [comment]
Just as a hint for testers (Beez doesn't load Bootstrap css always):
Deactivate this line before tests:
https://github.com/joomla/joomla-cms/blob/staging/templates/beez3/index.php#L41

to get the buttons styled correctly. Subforms don't look nice then but it is sufficient to test the concern of this pr.

**Comment by SharkyKZ** [comment]
I have tested this item :white_check_mark: successfully on db22fd0bdc9108b848364a4d195f50386c5e3fc3<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/20225">issues.joomla.org/tracker/joomla-cms/20225</a>.</sub>

**Comment by SharkyKZ** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/20225">issues.joomla.org/tracker/joomla-cms/20225</a>.</sub>

**Comment by HLeithner** [comment]
It took some time to merge but thanks for you fix.

**Pull request merged**